agent/plugins/metrics/prometheus/src/main/java/org/apache/shardingsphere/agent/metrics/prometheus/collector/BuildInfoCollector.java

```diff
@@ -32,23 +33,28 @@
  * Build information collector.
  */
 @Slf4j
+@AllArgsConstructor
 public final class BuildInfoCollector extends Collector {
     
     private static final String PROXY_BOOTSTRAP_CLASS = "org.apache.shardingsphere.proxy.Bootstrap";
     
     private static final PrometheusWrapperFactory FACTORY = new PrometheusWrapperFactory();
     
+    private final boolean isRunWithProxy;
```

Comment:
What is the different between `isEnhancedForProxy`  and `isRunWithProxy`

Reply:
I mean the same thing, let me adjust it.